book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,158 @@
+# Simple Payment and State Verification
+
+Thin clients using Solana require a receipt for transactions that have been
```

Comment:
This introduction can use more context. Thinking something like this:

The transaction contains a signature. Validators store those signatures for a short period of time to ensure the transaction is not processed more than once. Validators provide a JSON RPC endpoint, which thin clients can use to query the cluster if a transaction has been recently processed. Validators also provide a PubSub notification, whereby a thin client registers to be notified when a given signature is observed by the validator. While these two mechanisms allow a thin client to verify a payment, LIST_FAULTS_HERE.